runtime/jcl/common/system.c

```diff
@@ -140,10 +138,14 @@ Java_java_lang_System_getSysPropBeforePropertiesInitialized(JNIEnv *env, jclass
 	case 2: /* file.encoding */
 		sysPropValue = getDefinedEncoding(env, "-Dfile.encoding=");
 		if (NULL == sysPropValue) {
-#if JAVA_SPEC_VERSION >= 18
-			sysPropValue = "UTF-8";
-#else /* JAVA_SPEC_VERSION >= 18 */
+#if JAVA_SPEC_VERSION < 18
 			sysPropValue = getPlatformFileEncoding(env, property, sizeof(property), sysPropID);
+#else /* JAVA_SPEC_VERSION >= 18 */
```

Comment:
The comment here and on line 149 should match the `#if` condition (`JAVA_SPEC_VERSION < 18`).

Reply:
Done.